build.gradle

```diff
@@ -160,3 +161,5 @@ buildScan {
     licenseAgreementUrl = 'https://gradle.com/terms-of-service'
     licenseAgree = 'yes'
 }
+
+apply from: 'gradle/license.gradle'
```

Comment:
For clarity, is it possible to move this statement to the line where we apply other Gradle script plugins?